src/main/resources/db/migration/V1_90_correct_auth_user_id_refs.sql

```diff
@@ -0,0 +1,44 @@
+do 
+$$
+declare
+    sp_user record;
+begin
+	raise notice 'Migration to correct auth user id references - Started';
+    for sp_user in
+		select au.id id, au_d.id id_deleted from auth_user au 
+		inner join auth_user au_d on au_d.user_name = au.user_name and au_d.deleted is true
+		where au.auth_source = 'auth'
+		and au.deleted is not true
```

Comment:
should we do `au.deleted is false`?
I assume `is not true` will also return nulls.